tests/shared/robot.spec.js

```diff
@@ -22,6 +22,23 @@ const robot = require('../../lib/shared/robot');
 
 describe('Shared: Robot', function() {
 
+  describe('.COMMAND', function() {
+
+    it('should be a plain object', function() {
+      m.chai.expect(_.isPlainObject(robot.COMMAND)).to.be.true;
+    });
+
+    it('should only contain string values', function() {
+      m.chai.expect(_.every(_.values(robot.COMMAND), _.isString)).to.be.true;
+    });
+
+    it('should contain only unique values', function() {
+      const numberOfKeys = _.size(_.keys(robot.COMMAND));
+      m.chai.expect(_.size(_.uniq(_.values(robot.COMMAND)))).to.equal(numberOfKeys);
+    });
+
+  });
+
```

Comment:
Fixed! `robot.log` only prints to `stdout`, which is quite tricky to test. It internally uses `.buildMessage`, which is already well tested.